keyboards/handwired/ziyoulang_k3_mod/ziyoulang_k3_mod.c

```diff
@@ -0,0 +1,119 @@
+// Copyright 2023 Coom (@coomstoolbox)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+#include <stdio.h>
+
+void keyboard_post_init_kb(void) {
+#ifdef CONSOLE_ENABLE
+    debug_enable=true;
+    debug_matrix=true;
+#endif
+    keyboard_post_init_user();
+}
+
+static uint16_t last_keycode = KC_NO;
+static uint16_t last_col = 0;
+static uint16_t last_row = 0;
```

Comment:
both of these are uint8_t, and may be simpler to use that.  Also, you could use keypos_t, as that's just a structure with both col and row.